src/frontend/typer/oppgave.ts

```diff
@@ -149,17 +152,47 @@ export enum PrioritetFilter {
     LAV = 'Lav',
 }
 
-export enum DokumentType {
-    TEST,
+export enum Dokumenttype {
+    SØKNAD_OM_BARNETRYGD = 'SØKNAD_OM_BARNETRYGD',
 }
 
+export enum LogiskeVedleggTyper {
```

Comment:
Hvorfor er denne typen vedlegg kalt "logiske"? :)

Reply:
Fordi det er typene som fag ønsker for "annet innhold", men som heter logiske vedlegg hos dokument.